app/controllers/admin/payments_controller.rb

```diff
@@ -52,10 +52,11 @@ def update_transactions
   end
 
   def export_payments
+    payment_type = params[:payment_type] == "Payconiq" ? [:payconiq_online, :payconiq_display] : [:ideal]
```

Comment:
Why do we give an array with both payconiq types? Shouldn't we only have the online one?

Reply:
All payconiq transactions currently are paid in one transaction but this is irrelevant for now as long as the payconiq display has no functionality